internal/storage/v2/grpc/trace_storage.proto

```diff
@@ -62,6 +65,44 @@ message GetOperationsResponse {
   repeated Operation operations = 1;
 }
 
+// TraceQueryParameters contains query paramters to find traces. For a detailed
+// definition of each field in this message, refer to `TraceQueryParameters` in `jaeger.api_v3`
+// (https://github.com/jaegertracing/jaeger-idl/blob/main/proto/api_v3/query_service.proto).
+message TraceQueryParameters {
+  string service_name = 1;
+  string operation_name = 2;
+  repeated opentelemetry.proto.common.v1.KeyValue attributes = 3;
+  google.protobuf.Timestamp start_time_min = 4 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
+  google.protobuf.Timestamp start_time_max = 5 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
+  google.protobuf.Duration duration_min = 6 [
+    (gogoproto.stdduration) = true,
+    (gogoproto.nullable) = false
+  ];
+  google.protobuf.Duration duration_max = 7 [
+    (gogoproto.stdduration) = true,
+    (gogoproto.nullable) = false
+  ];
+  int32 search_depth = 8;
+}
+
+// FindTracesRequest represents a request to find traces.
+// It can be used to retrieve the traces (FindTraces) or simply 
+// the trace IDs (FindTraceIDs).
+message FindTracesRequest {
+  TraceQueryParameters query = 1;
+}
+
+// FindTraceIDsResponse represents the response for FindTracesRequest.
+message FindTraceIDsResponse {
+  repeated bytes trace_ids = 1;
```

Comment:
I just realized - shouldn't we be returning IDs with timestamps? We can do in a follow up PR, as it needs a change in storage API.